benchmarks/math/triangulate.gd

```diff
@@ -0,0 +1,28 @@
+extends Benchmark
+
+const RADIUS := 1.0
+const NUM_SIDES := 10_000
+const POSITION := Vector2(0.0, 0.0)
+
+# Generate a polygon circle of a certain `radius` with
+# a certain `num_side` and `position` as the center point.
+func generate_circle_polygon(radius: float,
+							 num_sides: int,
+							 position: Vector2) -> PackedVector2Array:
+	var angle_delta: float = (PI * 2) / num_sides
+	var vector: Vector2 = Vector2(radius, 0)
+	var polygon: PackedVector2Array
+
+	for _i in num_sides:
+		polygon.append(vector + position)
+		vector = vector.rotated(angle_delta)
+
+	return polygon
```

Comment:
Based on this [answer](https://ask.godotengine.org/81776/how-to-shape-polygon2d-into-a-circle?show=140975#a140975) from Godot's Q&A website.